site/pages/create-disruption.tsx

```diff
@@ -1,5 +1,211 @@
+                            <select className="govuk-select w-3/4" id="disruption-reason" name="disruptionReason">
+                                {Object.keys(Reason).sort().map(renderOption)}
+                            </select>
+                        </div>
+                    </fieldset>
+                </div>
+                <div className="govuk-form-group">
+                    <fieldset className="govuk-fieldset" role="group" aria-describedby="when-is-the-disruption">
+                        <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 className="govuk-fieldset__heading">When is the disruption?</h1>
+                        </legend>
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-4">
+                            What is the start date?
+                        </legend>
+                        <div id="start-date-hint" className="govuk-hint">
+                            Enter in format DD/MM/YYYY
+                        </div>
+                        <DateSelector
+                            errors={[]}
+                            startOrEnd="start"
+                            inputs={{
+                                dayInput: "",
+                                monthInput: "",
+                                yearInput: "",
+                            }}
+                        />
+
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-4">
+                            What is the start time?
+                        </legend>
+                        <div id="start-time-hint" className="govuk-hint">
+                            Enter in format xxxx
+                        </div>
+                        <TimeSelector
+                            errors={[]}
+                            startOrEnd="start"
+                            inputs={{
+                                hoursInput: "",
+                                minuteInput: "",
+                            }}
+                        />
+
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-4">What is the end date?</legend>
+                        <div id="end-date-hint" className="govuk-hint">
+                            Enter in format DD/MM/YYYY
+                        </div>
+                        <DateSelector
+                            errors={[]}
+                            startOrEnd="end"
+                            inputs={{
+                                dayInput: "",
+                                monthInput: "",
+                                yearInput: "",
+                            }}
+                        />
+
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-4">What is the end time?</legend>
+                        <div id="end-time-hint" className="govuk-hint">
+                            Enter in format xxxx
```

Comment:
Is this a helpful hint? If I'm told to enter in format xxxx, it seems a little unhelpful. HH:MM and an example would be good I think

Reply:
discussing with the design team and will update it appropriately

Reply:
resolved to be HH:MM @alvak1  changes are on my branch you can pull them